uaclient/cli.py

```diff
@@ -677,6 +677,16 @@ def action_config_set(args, cfg, **kwargs):
         }
         kwargs[set_key[4:]] = set_value
         setup_apt_proxy(**kwargs)
+    elif set_key in ("update_messaging_timer", "update_status_timer"):
+        try:
+            set_value = int(set_value)
+            if set_value < 0:
+                raise ValueError()
```

Comment:
I think we should have a message here saying why we raised the value error.
Furthermore, maybe it is a good to idea to also have this validation on the behave test